### DISCUSSION TITLE
加入开往维护团队，一起捍卫开放的互联网

### DISCUSSION THREAD

**Discussion by linlinzzo** [travellings-link/travellings#1747] [公告]
前一次维护开往已是2021年10月，开往加入缓慢、无效网站处理慢，可能一直是开往所困扰的。

> 我没足够的精力来维护这个项目了。——逊狼

为了解决审核和维护时间长的问题，我们决定组建维护团队，组织一批人来维护开往。

### 我加入维护团队要干什么？
1. 审核新加入的请求
2. 巡查开往的成员网站
3. 给成员网站评论冒泡

### 我加入团队需要什么？
条件待定。目前确定的是，你需要具有一颗适合团队协作的心。

### 我如何申请加入？
在本 issue 留言或通过邮箱 812568734@qq.com 联系我。

最后，用 readme 的一句话结尾：

> 网络曾经从封闭走向开放，如今又走向了封闭。在此环境下，独立网页（类似“独立游戏”）的流量或多或少的受到影响。开往尝试让传统友链“活跃”，让网页相互接力，让流量相互流动，让网络开放起来。

**Comment by imjason** [comment]
我希望可以加入团队

**Comment by HAHH9527** [comment]
申请加入维护团队

**Comment by Jiaocz** [comment]
申请加入~

<img src="https://static.orangii.cn/stickers/AmashiroNatsukiEars/13.webp" height="50px" />

**Comment by PipecraftNet** [comment]
申请加入维护团队

**Comment by linlinzzo** [comment]
> 申请加入维护团队

加入申请驳回。

**Comment by imjason** [comment]
> 申请加入维护团队

由于一些客观原因，开往团队暂时不需要更多成员了，谢谢你的申请

**Comment by wznmickey** [comment]
看到这个issue reopen了，如果是还需要招人的话，申请加入维护团队

**Comment by MaxwellLei** [comment]
如果是还需要招人的话，申请加入维护团队

**Comment by jiangyangcreate** [comment]
还需要1名全栈工程师熟悉Python/nodejs，有意向可以联系我们。@travellings-link/it
